xbmc/music/Artist.h

```diff
@@ -29,6 +29,15 @@ class CDiscoAlbum
   std::string strReleaseGroupMBID;
 };
 
+struct ArtistVideoLinks
+{
+public:
```

Comment:
No need for `public:`here as this is the default for structs. You can remove this line.

Reply:
Done.